app/main.py

```diff
@@ -1,3 +1,69 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int | float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: Distance) -> Distance:
+
+        if isinstance(other, Distance):
+            return Distance(
+                self.km + other.km
+            )
+        else:
+            self.km += other
+            return self
+
+    def __iadd__(self, other: int | float) -> Distance:
+        if isinstance(other, Distance):
+            self.km += other.km
+        else:
+            self.km += other
+
+        return self
+
+    def __mul__(self, other: int | float) -> Distance:
+        self.km *= other
+        return self
+
+    def __truediv__(self, other: int | float) -> Distance:
+        return Distance(
+            round(self.km / other, 2)
+        )
+
+    def __lt__(self, other: Distance) -> bool:
+        if isinstance(other, Distance):
+            return self.km < other.km
+        else:
+            return self.km < other
+
+    def __gt__(self, other: Distance) -> bool:
+        if isinstance(other, Distance):
+            return self.km > other.km
+        else:
+            return self.km > other
```

Comment:
1) `else` redundant here
2) add additional info for annotation 
fix it here and other places 

Example:
```suggestion
    def __gt__(self, other: Distance | int | float) -> bool:
         if isinstance(other, Distance):
             return self.km > other.km
         return self.km > other
```